R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2070,6 +2072,16 @@ test_that("dapply() on a DataFrame", {
   rownames(expected) <- NULL
   expect_identical(expected, result)
 
+  result <- dapplyCollect(
+              df,
+              function(x) {
+                y <- x[x$a > 1, ]
+                y <- cbind(y, y$a + 1L)
```

Comment:
@NarineK, not sure if I understand. the user input R function should return data.frame.